packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -132,3 +110,41 @@ export const SelectedItemCheck = styled( Ariakit.SelectItemCheck )`
 	margin-inline-start: ${ ITEM_PADDING };
 	font-size: 24px; // Size of checkmark icon
 `;
+
+const truncateStyles = css`
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+`;
+
+export const ItemHintWrapper = styled.div`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
```

Comment:
Added these two rules to allow hint to wrap, and to preserve vertical centering even after adding a smaller `line-height` for the hint